package.json

```diff
@@ -40,7 +40,7 @@
   },
   "dependencies": {
     "multibase": "^3.0.1",
-    "multicodec": "^2.0.1",
+    "multicodec": "=2.0.4",
```

Comment:
Can't we just do "2.0.4"?

Reply:
I didn't know that, but with out without the `=` seems to do exactly the same (I checked with https://semver.npmjs.com/). The equal sign makes it more explicit, which I like.